### ISSUE TITLE
Purchase Button is not Visible on Extension page

### ISSUE THREAD

**Issue by Talamantez** [Talamantez/claude-vscode#19]
When a user clicks the 'purchase now' button from VSCode, it takes them to the extension's page, but there is no visible purchase option.

I've opened a ticket with Microsoft support.

**Comment by Talamantez** [comment]
Update: In communication with support - they are reviewing V1.6.

I'll have an update ASAP.  Users may begin dropping off after a 7 day trial without being able to purchase.

**Comment by Talamantez** [comment]
Update: Notified support that I pushed v2.1.7

**Comment by Talamantez** [comment]
I am still waiting on support - I'm going to open a new ticket. Rest assured, we'll address this issue ASAP - 

**TO USERS** Please reach out if you have any questions and please open an issue if you run into any issues during the transition as the trial expires.

Thank you for your patience while we address this critical issue.

**Comment by Talamantez** [comment]
**NOTE**:  If this issue is not resolved soon, I will extend the trial. Thank you!

**Comment by Talamantez** [comment]
Support got back to me - They let me know that I can't actually monetize on the marketplace, so I am removing the trial version and open-sourcing the extension.
